<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReturnBook</class>
 <widget class="QWidget" name="ReturnBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Return Book</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>521</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="return_title_lbl">
      <property name="text">
       <string>Book Title: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="title_lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="return_btn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Return Book</string>
   </property>
  </widget>
  <widget class="QLabel" name="return_msg_lbl">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>260</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
